### DISCUSSION TITLE
UIButton.Configuration.image 사이즈 문제

### DISCUSSION THREAD

**Discussion by nomatterjun** [Favor-Gift-Reminder/Favor-iOS#35] [Q&A] [answered]
`UIButton.Configuration`을 사용해서 버튼에 이미지를 넣으려고 하는데..
이미지 사이즈를 바꾸는 법을 모르겠네.. 😭
혹시 아는 방법 있을까..?

``` swift
config.contentInsets = .init(top: 10, leading: 10, bottom: 10, trailing: 10)
```

``` swift
config.imagePadding = 10
```

``` swift
config.preferredSymbolConfigurationForImage = .init(pointSize: 17.0)
```

이렇게 해봤는데 안되네.. 😅

**Comment by EnnzoKim** [comment, marked as answer]
음... 나도 찾아봤는데 버튼 자체의 사이즈를 줄이는 방법은 잘 안나오네 ..ㅠㅠ

혹시 어떤 경우에 버튼의 사이즈를 바꾸는 건지 알 수 있을까?

만약 버튼의 사이즈를 바꾸고 싶으면
그 에셋의 이미지를 `Large`, `medium` `small`이런 식으로 각각의 사이즈를 따로 넣는게 낫지 않을까? 라는 생각이야!

**Reply by nomatterjun** [reply]
![스크린샷 2023-01-21 11 36 15](https://user-images.githubusercontent.com/60438045/213840166-ad6b4cf4-20ba-4851-b167-6d0d8e44047f.png)

프로필 추가 부분에서 아이콘 사이즈가 안늘어나더라고..
에셋 사이즈를 따로 넣는 것도 방법이겠다..!

고민을 더 해볼게!

**Reply by EnnzoKim** [reply]
응응 사이즈를 직접 변경하기보다는 
차라리 원본 이미지를 사용하는게 나으니까 
에셋 사이즈를 따로 넣는게 나을 것 같아! 

`ic_human_small`
`ic_human_large` 이런식으로!
